Sources/SQLKit/Builders/SQLAlterTableBuilder.swift

```diff
@@ -19,6 +19,21 @@ public final class SQLAlterTableBuilder: SQLQueryBuilder {
         self.database = database
     }
     
+    @discardableResult
```

Comment:
Could you add some API docs for this? I know the rest of the class doesn't have this but we should make sure we try and add them as we go forward

Reply:
Sure, no problem. I've added API docs throughout this file for good measure.